core/validation/specific_validation/pathfinders/round_1.py

```diff
@@ -0,0 +1,163 @@
+import pandas as pd
+
+from core.exceptions import ValidationError
+from core.messaging import Message
+from core.transformation.pathfinders.round_1.control_mappings import (
+    create_control_mappings,
+)
+
+
+def cross_table_validation(tables: dict[str, pd.DataFrame]) -> None:
+    """
+    :param tables: set of tables to validate
+    """
+
+    # TODO add correct error messages from design and error types
+    mappings = create_control_mappings(tables)
+
+    tamper_checks = (_check_projects, _check_standard_outputs_outcomes, _check_bespoke_outputs_outcomes)
+    validation_checks = _check_financial_underspend_values
+
+    for check in tamper_checks:
+        check(tables, mappings)
+
+    messages = []
+    for check in validation_checks:
+        messages.extend(check(tables, mappings))
+
+    if messages:
+        raise ValidationError(messages)
+
+
+def _check_projects(input_tables: dict[str, pd.DataFrame], control_mappings) -> None:
+    """
+    :param tables: set of tables to validate
+    :param mappings: the control mapping which we validate input dataframes against
+    """
+
+    # The organisation used by the user
+    table_names = ["Project progress", "Project location"]
+
+    organisation_name = input_tables["Organisation Name"].df.iat[0, 0]
```

Comment:
Should be `input_tables["Organisation name"]`. All extracted tables are uppercase first letter then all lowercase